storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/dataaccess/CombinedKvStoreDao.java

```diff
@@ -95,6 +95,11 @@ public Optional<SignedBeaconBlock> getHotBlock(final Bytes32 root) {
     return db.get(schema.getColumnHotBlocksByRoot(), root);
   }
 
+  @Override
+  public Optional<Bytes> getHotBlockAsSsz(Bytes32 root) {
```

Comment:
final

Reply:
ye I copy\pasted what I had in the surroundings...